README.md

```diff
@@ -153,6 +154,31 @@ The PER for this model is:
 
 We provide a Colab notebook to run a pre-trained TDNN LiGRU CTC model:  [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/11IT-k4HQIgQngXz1uvWsEYktjqQt7Tmb?usp=sharing)
 
+### TED-LIUM3
+
+We provide two models for this recipe: [Transducer Stateless: Conformer encoder + Embedding decoder][TED-LIUM3_transducer_stateless] and [Pruned Transducer Stateless: Conformer encoder + Embedding decoder + k2 pruned RNN-T loss][TED-LIUM3_pruned_transducer_stateless].
+
+#### Transducer Stateless:  Conformer encoder + Embedding decoder
+
+The best WER using modified beam search with beam size 4 is:
+
+|     |  dev  |  test  |
+|-----|-------|--------|
+| WER |  6.91 |  6.33  |
+
+Note: No auxiliary losses are used in the training and no LMs are used in the decoding.
+
+We provide a Colab notebook to run a pre-trained Transducer Stateless model: [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1MmY5bBxwvKLNT4A2DJnwiqRXhdchUqPN?usp=sharing)
```

Comment:
I find that this colab notebook is for librispeech. Could you update it to point to the notebook for tedlium3?

Reply:
I plan to update it for tedlium3 after merging. Because it will be better to do things such as `git clone https://github.com/k2-fsa/icefall` or `cd icefall/egs/tedlium3/ASR`.

Reply:
You can first use
```bash
git clone https://github.com/luomingshuang/icefall
cd icefall
git checkout tedlium3-pruned-transducer-stateless-new
```
in the notebook.

After merging, you can replace the above 3 lines with

```bash
https://github.com/k2-fsa/icefall
cd icefall
```